### ISSUE TITLE
Does it make sense for toJSON() to include computed properties?

### ISSUE THREAD

**Issue by d3chapma** [skaterdav85/backbone-computed-properties#8]
It would seem to me that if we are computing the properties based on other properties that we don't want to send them to the server.

**Comment by iamdtang** [comment]
correct. I had to override `toJSON` a lot. Never implemented an automated way to ignore the computed properties.

**Comment by iamdtang** [comment]
I don't work with Backbone anymore, but I can see this being a good addition. If you'd like to add this, feel free to do so. Maybe a good approach would be to make this an opt-in so it doesn't break existing users who might be using computed properties from toJSON(). If you want to add it, let me know and I'll keep this issue open. Otherwise I will close it.
